README.md

```diff
@@ -35,3 +36,7 @@ Checkout the [spec](https://drive.google.com/file/d/1_WuEODpEKV7BR3LMKVLxyZUWjcG
       autonomy packages sync
 
 - During development use `make formatters`, `make code-checks` and `make generators`
+
+- After building your development environment and pulling the packages, you can conduct an end-to-end test with a local network and 4 agents by running
+      
+      pytest packages/balancer/agents/autonomous_fund/tests/test_agents/test_autonomous_fund.py::TestAutonomousFundFourAgents
```

Comment:
Why not use `autonomy test by-path`?

Reply:
There are multiple e2e test, and contract tests. I am not aware of a way to run a single one via the aea command.

Reply:
ok this will do for now. Although you could add a new line and say the other command and that it will run all tests there.